### ISSUE TITLE
request: noone, someone, everybody

### ISSUE THREAD

**Issue by andyinno** [home-assistant/appdaemon#263]
from the documentation of noone_home appdaemon uses the state of group.all_devices()
for getting it's state. 
If you are enabling a discovery the known_devices file gets filled with a lot of devices that you don't really want to track or that are reduntant or just not needed for the meaning of the function.

For example:
I would like to track my tablet for having homeassistant to alert me if the battery is running low but I leave it at home if I don't need it.
I would like to track the bluetooth speaker in order to know if it is turned on or off.

In my installation I use a group for tracking a person. This because sometimes the single tracked entity can give false positive/negative. GPS approximation, ping timeout etc.

I think that it would be nice and useful to have noone_home() to track entities specified in the configuration file. It can give the flexibility to add devices or entities in general that reflect what the user like.

something like:

``` 
home_status:
  - device_tracker.bluetooth_phone
  - group.my_user
``` 

What do you think about it? Am I missing something about the meaning of the function?
thank you
Andrea

**Comment by acockburn** [comment]
Actually noone_home() checks the state of all device trackers in the system rather than looking at group.all_devices - I found this more reliable in avoiding race conditions.

What you are asking for is very easily coded in an app, and I try and avoid breaking changes where possible.

Having said that, I'm unhappy about the way device trackers work in general (from a HASS perspective) so something like this might be a way to improve on things.

**Comment by andyinno** [comment]
Hi @acockburn 

first of all, thank you for your kind reply.

I completely understand and approve your second and third sentence. It is of course easily coded in an app and the way home assistant uses the device tracking sucks. 
This is the reason I ended up making an example before that uses a group. I ended up adding multiple group defining different device trackers, a family group (everyone home) (someone home) and I am looking for some viable alternative to it. Then I was looking at appdaemon documentation and I found the noone home method but it looks like it is not what I need, and in my opinion it is not really useful because it gets flooded with all the devices that someone may have it his own house.

Writing the app myself it is not a problem. I can do it. I am just pointing to the fact that it could be useful if appdaemon have it included by default, or as a plugin.
I never read deeply how appdaemon internal works, but if you think this could be a good plugin I can contribute, with some guidance, instead of writing my own app.

thanks again
Andrea
